### ISSUE TITLE
Plese change the icon to the addon icon instead

### ISSUE THREAD

**Issue by secretmango** [jscher2000/cache-favicons-for-bookmarks-extension#6]
The URL bar is not editable. Currently I have way too many icons there which I never use.

Meanwhile the normal Addon icons with the same features can be hidden or unhidden.

The addon works great, but I never manually change anything. It would be great if it was just there and not clutter the space.

Especially when having more interactive Addons like 

- Dark Background light text
- UBlock
- Noscript
- Tab groups
- ...

These have to be shown.

In the URL bar addons like

- QR
- FF Translate
- County Flags & IP Whois
- FF multi account containers

already clutter the space.

I think this is poor design, and makes it pretty much unusable when tiling the browser window in half.

**Comment by scottewing-games** [comment]
I agree that the icon that sits in the URL bar is a nuisance. It seems to me that the functionality of this app could easily be managed through an icon in the addons area of the toolbar, instead of in the URL bar itself, which would then allow for the icon to be hidden when not being used. An update for this reason would be appreciated.

**Comment by secretmango** [comment]
doesnt look like the dev is active anymore.

**Comment by jscher2000** [comment]
Sorry this took so long. Should be live for you in version 0.7.

<img width="735" height="255" alt="Image" src="https://github.com/user-attachments/assets/b8f4cea4-3d08-48f4-91a5-ddd5e7a7dc2a" />
